datastore.yml

```diff
@@ -0,0 +1,16 @@
+---
+- hosts: datastore-dev
+  remote_user: root
+  gather_facts: no
+  roles:
+    - common/python2
+    - common/postgres
+    - common/create_user
+    - dev_datastore/datastore_setup
```

Comment:
Is this for all `datastore` instances rather than just a `dev datastore`?

Reply:
datastore